bug: Non-Deterministic Test KafkaServiceBootstrapReconcilerIT.shouldUpdateStatusOnceTrustAnchorConfigMapDeleted

A test failed non-deterministically in GitHub Actions:

## failed test

`io.kroxylicious.kubernetes.operator.reconciler.kafkaservice.KafkaServiceBootstrapReconcilerIT.shouldUpdateStatusOnceTrustAnchorConfigMapDeleted`
https://github.com/kroxylicious/kroxylicious/actions/runs/29629011737/job/88039698200

## stack trace

```
[ERROR] io.kroxylicious.kubernetes.operator.reconciler.kafkaservice.KafkaServiceBootstrapReconcilerIT.shouldUpdateStatusOnceTrustAnchorConfigMapDeleted -- Time elapsed: 0.212 s <<< ERROR!
java.lang.NullPointerException: Cannot invoke "io.kroxylicious.kubernetes.api.v1alpha1.KafkaServiceStatus.getBootstrapServers()" because the return value of "io.kroxylicious.kubernetes.api.v1alpha1.KafkaService.getStatus()" is null
	at io.kroxylicious.kubernetes.operator.reconciler.kafkaservice.KafkaServiceBootstrapReconcilerIT.lambda$assertResolvedRefsTrue$1(KafkaServiceBootstrapReconcilerIT.java:290)
	at org.awaitility.core.AssertionCondition.lambda$new$0(AssertionCondition.java:53)
	at org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:248)
	at org.awaitility.core.ConditionAwaiter$ConditionPoller.call(ConditionAwaiter.java:235)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
```

--- a/kroxylicious-kubernetes/kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/reconciler/kafkaservice/KafkaServiceBootstrapReconcilerIT.java
+++ b/kroxylicious-kubernetes/kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/reconciler/kafkaservice/KafkaServiceBootstrapReconcilerIT.java
@@ -287,12 +287,15 @@ class KafkaServiceBootstrapReconcilerIT {
         AWAIT.untilAsserted(() -> {
             final KafkaService kafkaService = clusterUser.get(KafkaService.class, ResourcesUtil.name(cr));
             Assertions.assertThat(kafkaService).isNotNull();
-            Assertions.assertThat(kafkaService.getStatus().getBootstrapServers()).isEqualTo(expectedBootstrap);
             assertThat(kafkaService.getStatus())
                     .isNotNull()
-                    .conditionList()
-                    .singleElement()
-                    .isResolvedRefsTrue();
+                    .satisfies(status -> {
+                        Assertions.assertThat(status.getBootstrapServers()).isEqualTo(expectedBootstrap);
+                        assertThat(status)
+                                .conditionList()
+                                .singleElement()
+                                .isResolvedRefsTrue();
+                    });
             String checksum = getReferentChecksum(kafkaService);
             if (hasReferents) {
                 Assertions.assertThat(checksum).isNotEqualTo(NO_CHECKSUM_SPECIFIED);

